### PR TITLE
OSDOCS#5794: Migrate all the AWS Load Balancer Operator examples to versio "v1"

### PR DESCRIPTION
**Pull request by xenolinux**
Migrate the ALBO examples to new OLM channel stable-v1 and version v1.

Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OSDOCS-5794

Link to docs preview: https://59054--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.html#nw-creating-instance-aws-load-balancer-controller_create-instance-aws-load-balancer

https://59054--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.html#nw-creating-multiple-ingress-through-single-alb_multiple-ingress-through-single-alb

https://59054--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/installing-albo-sts-cluster.html#nw-installing-albo-on-sts-cluster-predefined-credentials_albo-sts-cluster

https://59054--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/install-aws-load-balancer-operator.html#nw-installing-aws-load-balancer-operator_aws-load-balancer-operator

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
@lihongan Can you please take a look at this PR?

File: modules/creating-multiple-ingress-through-single-alb.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ You can route the traffic to multiple Ingresses through a single AWS Load Balanc
 +
 [source,yaml]
 ----
-apiVersion: elbv2.k8s.aws/v1beta1 <1>
+apiVersion: elbv2.k8s.aws/v1 <1>
```
Comment (alebedev87) [inline modules/creating-multiple-ingress-through-single-alb.adoc:21]: Only `AWSLoadBalancerController` API was promoted to `v1`, the other APIs remained in their verions.

**Comment by alebedev87** [comment]
A mistake in the example from [this chapter](https://59054--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/add-tls-termination.html#nw-adding-tls-termination_adding-tls-termination) - `apiVersion` and `group` should be replaced by just `apiVersion`:
```
apiVersion: networking.olm.openshift.io/v1
```

**Comment by lihongan** [comment]
looks good. Thank you @xenolinux

**Pull request merged**